libs/live-config/src/FeatureFlags/useFeature.test.tsx

```diff
@@ -1,6 +1,7 @@
 import { renderHook } from "@testing-library/react-hooks";
 import useFeature from "./useFeature";
 import { makeMockedFeatureFlagsProviderWrapper, makeMockedContextValue } from "./mock";
+import { describe, it, expect } from "@jest/globals";
```

Comment:
Should not be needed if you update the jest config (see other comment in `jest.config.js`

```suggestion
```

Reply:
fixed